### PR TITLE
cmd/destroy: improve destroy cmd

### PR DESCRIPTION
**Pull request by zhijian-pro**

### CODE REVIEW

File: cmd/destroy.go [hunk 1]
```diff
@@ -135,7 +135,9 @@ func destroy(ctx *cli.Context) error {
 			for i, s := range sessions {
 				ss[i] = [3]string{strconv.FormatUint(s.Sid, 10), s.HostName, s.MountPoint}
 			}
-			logger.Fatalf("%d sessions are active, please disconnect them first:\n%s", num, printSessions(ss))
+			logger.Infof("%d sessions are active, please disconnect them first or with --force param skip sanity check and force destroy the volume:\n", num)
+			fmt.Println(printSessions(ss))
+			logger.Exit(0)
```
Comment (SandyXSD) [inline cmd/destroy.go:140]: Exit(1)

File: cmd/destroy.go [hunk 2]
```diff
@@ -135,7 +135,9 @@ func destroy(ctx *cli.Context) error {
 			for i, s := range sessions {
 				ss[i] = [3]string{strconv.FormatUint(s.Sid, 10), s.HostName, s.MountPoint}
 			}
-			logger.Fatalf("%d sessions are active, please disconnect them first:\n%s", num, printSessions(ss))
+			logger.Errorf("%d sessions are active, please disconnect them first:", num)
+			fmt.Println(printSessions(ss))
```
Comment (davies) [inline cmd/destroy.go:139]: Why?

Comment (zhijian-pro) [reply cmd/destroy.go:139]: image

Comment (davies) [reply cmd/destroy.go:139]: It's not a big deal. After this PR, the list of session will be in STDOUT, but others in STDERR.

**Pull request closed without merging**